doc/backing-store-crd.md

```diff
@@ -16,172 +22,208 @@ Adding support for a new type of backing-store is rather easy as it requires jus
+noobaa backingstore create ibm-cos bs --endpoint <> --access-key <> --secret-key <> --target-bucket <>
 ```
 ```yaml
 apiVersion: noobaa.io/v1alpha1
 kind: BackingStore
 metadata:
   finalizers:
   - noobaa.io/finalizer
-  labels:
-    app: noobaa
-  name: bs
-  namespace: noobaa
+  name: <>
+  namespace: <>
 spec:
   s3Compatible:
-    endpoint: ENDPOINT
+    endpoint: <>
     secret:
-      name: backing-store-ibm-cos-bs
-      namespace: noobaa
-    signatureVersion: v2
-    targetBucket: BUCKET
+      name: <>
+      namespace: <>
+    signatureVersion: <>
+    targetBucket: <>
   type: ibm-cos
 ```
 
-#### GOOGLE-CLOUD-STORAGE type
-
-Create a cloud resource within the NooBaa brain and use Google Cloud Storage API for storing encrypted chunks of data in Google Cloud Storage.
+## Google Cloud Storage
+Uses the Google Cloud Storage API for storing encrypted chunks of data in Google Cloud buckets
 ```shell
-noobaa -n noobaa backingstore create google-cloud-storage bs --private-key-json-file key.json --target-bucket BUCKET
+noobaa backingstore create google-cloud-storage <BACKINGSTORE NAME> --private-key-json-file <PATH TO credentials.json> --target-bucket <>
 ```
 ```yaml
 apiVersion: noobaa.io/v1alpha1
 kind: BackingStore
 metadata:
   finalizers:
   - noobaa.io/finalizer
-  labels:
-    app: noobaa
-  name: bs
-  namespace: noobaa
+  name: <>
+  namespace: <>
 spec:
   googleCloudStorage:
     secret:
-      name: backing-store-google-cloud-storage-bs
-      namespace: noobaa
-    targetBucket: BUCKET
+      name: <>
+      namespace: <>
+    targetBucket: <>
   type: google-cloud-storage
 ```
 
-#### AZURE-BLOB type
-
-Create a cloud resource within the NooBaa brain and use BLOB API for storing encrypted chunks of data in Azure cloud.
+## Azure Blob Storage
+Uses the Azure Blob API for storing encrypted chunks of data in an Azure container
 ```shell
-noobaa -n noobaa backingstore create azure-blob bs --account-key KEY --account-name NAME --target-blob-container CONTAINER
+noobaa backingstore create azure-blob <BACKINGSTORE NAME> --account-key <> --account-name <> --target-blob-container <>
 ```
 ```yaml
 apiVersion: noobaa.io/v1alpha1
 kind: BackingStore
 metadata:
   finalizers:
   - noobaa.io/finalizer
-  labels:
-    app: noobaa
-  name: bs
-  namespace: noobaa
+  name: <>
+  namespace: <>
 spec:
   azureBlob:
     secret:
-      name: backing-store-azure-blob-bs
-      namespace: noobaa
-    targetBlobContainer: CONTAINER
+      name: <>
+      namespace: <>
+    targetBlobContainer: <>
   type: azure-blob
 ```
 
-#### PV-POOL type
+## Persistent Volume Pool
+Creataes a StatefulSet with a PVC mounted in each pod. Each resource will connect to the NooBaa core and provide the PV filesystem storage to be used for storing encrypted chunks of data. It is possible to configure the number of pods to be used and their PV size.
```

Comment:
`Creataes` probably a spelling mistake ? 
We should have here an intro like the other backingstore types have. The first sentence is misplaced and the reader can't understand the purpose of this type of backingstore IMHO.

Reply:
much better